### PR TITLE
Enforcing 1000 wei min change

### PR DESCRIPTION
**Pull request by wtj2021**

### CODE REVIEW

**Comment by dapp-whisperer** [comment]
# Review Notes (WIP)
- MIN_CHANGE constant
- Introduces constant for new error message rather than inline string (like)

elegantly adapts checks for nonzero values to minimum values

```solidity
_requireNonZeroDebtChange → _requireMinDebtChange
_requireNonZeroAdjustment → _requireMinAdjustment
_requireNonZeroDebt -> _requireMinDebt
```

error messages adapt to new reality

## Open

- require debt is >1000 units

## Adjust

- on debt increase, confirm increase is >1000
- on debt decrease, confirm min debt is >1000 after change
- on coll change, confirm the stEthBalance (not shares) change is >1000 units
    - (min stEthBalance enforced as before)

## Test Change Notes

Introduces constants for new error message in fixture

confirms revert w/ correct error message, with 1 wei changes in-line in some existing tests

should confirm that all values  <1000 result in revert via fuzz tests when complete

- open
- adjust
- close
- partialLiquidation
- partialRedemption

**Comment by dapp-whisperer** [comment]
## Redemption Spec

Gracefully skip partials when CDP has < MIN_CHANGE debt remaining, similar to min coll check.
image

**Comment by wtj2021** [comment]
medusa coverage report https://drive.google.com/file/d/1lSy8wQWgwQK7RlBXSkprNnSGufUmTvVu/view?usp=drive_link

File: packages/contracts/contracts/LiquidationLibrary.sol [hunk 1]
```diff
@@ -916,4 +920,8 @@ contract LiquidationLibrary is CdpManagerStorage {
             "LiquidationLibrary: Coll remaining in partially liquidated CDP must be >= minimum"
         );
     }
+
+    function _requireMinDebt(uint256 _debt) internal pure {
```
Comment (dapp-whisperer) [inline packages/contracts/contracts/LiquidationLibrary.sol:924]: dup'd function (besides error message) but I'm not concerned about it.

**Pull request merged**